deny.toml

```diff
@@ -66,6 +66,7 @@ skip = [
   { name = "pulldown-cmark" },   # Build-dependency via `ply-rs` (!). TODO(emilk): use a better crate for .ply parsing
   { name = "redox_syscall" },    # Plenty of versions in the wild
   { name = "rustc-hash" },       # numpy with compatible pyo3 requires different version than wgpu
+  { name = "bzip2" },            # Datafusion-avro has us indirectly pinned to this
```

Comment:
where does the conflict stem though from? Do we oureslves just have to update bzip2 or is it another package that wants a different version?